### ISSUE TITLE
Details on calculating FID score

### ISSUE THREAD

**Issue by htwang14** [huawei-noah/GAN-pruning#4]
Hi,
Could you please provide more information on how you calculated the FID score? For example, which code did you use? How many images did you use to find FID score? 
Thanks in advance.

**Comment by shuh15** [comment]
please refer this [link](https://github.com/mseitzer/pytorch-fid) . We used transfer results of all the test data of horse2zebra, yosemite_summer2winter to calculate FID score.

**Comment by htwang14** [comment]
Thanks for your reply. Just to make sure, for the summer to winter task, are you calculating the distance between transfer results of summer test set and winter training set for FID?

**Comment by shuh15** [comment]
distance between transfer results of summer test set and winter test set.
